pkg/agent/interfacestore/types.go

```diff
@@ -71,6 +71,8 @@ type TunnelInterfaceConfig struct {
 	LocalIP net.IP
 	// IP address of the remote Node.
 	RemoteIP net.IP
+	// Port of the Tunnel.
+	Port ovsconfig.TunnelPort
```

Comment:
Don't the following `DestinationPort` mean the port of the tunnel?

Reply:
Thanks for the review, updated.